core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -426,4 +426,11 @@ package object config {
     .toSequence
     .createOptional
 
+  private[spark] val MAX_REMOTE_BLOCK_SIZE_TO_MEM =
```

Comment:
Shall consolidate this config and `REDUCER_MAX_REQ_SIZE_SHUFFLE_TO_MEM`? shuffle blocks and remote RDD blocks are all remote blocks.

Reply:
I was thinking about this, but the configuration name of `REDUCER_MAX_REQ_SIZE_SHUFFLE_TO_MEM` seems too shuffle specific, maybe we should rename it.

Reply:
how about `MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM` and `spark.executor.maxRemoteBlockSizeFetchToMemory`?

Reply:
I would prefer to use `spark.storage.maxRemoteBlockSizeFetchToMemory`, since driver side block manager will also leverage this feature.

Reply:
`spark.storage.maxRemoteBlockSizeFetchToMemory` is not very clear that it works for shuffle too...

Reply:
maybe just `spark.maxRemoteBlockSizeFetchToMemory`?